recipes/laspy/meta.yaml

```diff
@@ -0,0 +1,70 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "laspy" %}
+{% set version = "1.5.0" %}
+{% set sha256 = "9561d6d4cfb5ebfa8201151415d45085" %}
```

Comment:
I believe the sha256 hash should be:
329dc86a1296996a22d110ea8c155de8567089f5a9e5c7f02843725cc28cd3ab